features/dropdowns/relevance_default.feature

```diff
@@ -0,0 +1,9 @@
+  Feature: Default sort is relevance
+    In order to identify the most relevant results
+    As a researcher
+    I wan the default sort option to be Relevance
+
+  Scenario: Default sort option is Relevance
+    Given I am on the default search page
+    When I search on the phrase "bloch"
+    Then "Sort by relevance" should be selected for "sort" dropdown
```

Comment:
@chrpr Sorting is only on the brief results screen so you could simplify assuming you're always on the brief results screen and then each story could be 

```
Given I am on the brief results screen
When I select "relevance" from the "Sort by" dropdown 
Then the results should be sorted by relevance
```

Where the last statement is just a feature-pointed way of checking that the dropdown is correctly selected.

Thoughts?